src/test/java/school/cesar/eta/unit/PersonTest.java

```diff
@@ -1,62 +1,132 @@
 package school.cesar.eta.unit;
 
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Random;
+
 import static org.junit.jupiter.api.Assertions.fail;
 
 public class PersonTest {
+    Person a;
+    @BeforeEach
+    public void setup(){
+        a = new Person();
+    }
+
     @Test
     public void getName_firstNameJonLastNameSnow_jonSnow() {
-        fail();
+        a.setName("Jon");
+        a.setLastName("Snow");
+
+        Assertions.assertEquals("Jon Snow" , a.getName());
     }
 
     @Test
     public void getName_firstNameJonNoLastName_jon() {
-        fail();
+        a.setName("Jon");
+
+        Assertions.assertEquals("Jon" , a.getName());
     }
 
     @Test
     public void getName_noFirstNameLastNameSnow_snow() {
-        fail();
+        a.setLastName("Snow");
+
+        Assertions.assertEquals("Snow", a.getName());
     }
 
     @Test
     public void getName_noFirstNameNorLastName_throwsException() {
-        fail();
-    }
 
-    @Test
-    public void isBirthdayToday_differentMonthAndDay_false() {
-        fail();
+        Assertions.assertThrows(RuntimeException.class,() -> new Person().getName() );
     }
 
+    //@Test
+    //public void isBirthdayToday_differentMonthAndDay_false() { fail(); }
+
     @Test
     public void isBirthdayToday_sameMonthDifferentDay_false() {
-        fail();
+        ArrayList<String> listNum = new ArrayList<String>();
+        for (int x = 1; x <= 9; x++) {
+            listNum.add(String.valueOf(x));
+        }
+
+        listNum.remove(String.valueOf(LocalDate.now().getDayOfMonth()));
+        Random y = new Random();
+        int day = y.nextInt(7);
+
+        //LocalDate dataManipulacao = LocalDate.now();
+        //dataManipulacao.plusDays(5).plusMonths(1);
+
+        String date = LocalDate.now().getYear() + "-" + LocalDate.now().getMonthValue() + "-0" +  listNum.get(day);
```

Comment:
A data retornada pelo `LocalDate.now()` é calculado no momento em que o método é chamado, então por frações de segundo o dia, mês e ano calculado podem variar entre uma chamada e outra, o que pode provocar intermitências no resultado do teste.

**Evite usar números randômicos e basear seus testes em variáveis do sistema!**

Para isso,  use um valore imutável em `setBirthday()` e crie um [stub](https://spring.io/blog/2007/01/15/unit-testing-with-stubs-and-mocks) do método getNow() retornando uma data fixa.